lib/diagnostic.js

```diff
@@ -19,15 +19,15 @@
 //
 //     `true` or `false`
 //
-let access = null; // replace null with your response
+let access = false; // replace null with your response
 //
 
 // Which scope(s) may access the variable `laserQuantity`? Choose one or many
 // of the following:
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-let scopes = [/* your response here */];
+let scopes = [deathStar & fire];
```

Comment:
Yes! But put these as strings in an array, so that the tests pass.
`['deathStar', 'fire']`